Backend/app/rider-platform/rider-app/Main/src/API/Dashboard/RideBooking/Confirm.hs

```diff
@@ -54,4 +55,8 @@ handler =
 callConfirm :: ShortId DM.Merchant -> Id DP.Person -> Id Quote.Quote -> Maybe (Id DMPM.MerchantPaymentMethod) -> FlowHandler UC.ConfirmRes
 callConfirm merchantId personId quote mbPaymentMethodId = do
   m <- withFlowHandlerAPI $ findMerchantByShortId merchantId
-  UC.confirm (personId, m.id) quote mbPaymentMethodId
+  res <- UC.confirm (personId, m.id) quote mbPaymentMethodId
+  withFlowHandlerAPI $ do
+    fork "sending Booking confirmed dasboard sms" $ do
+      Sms.sendDashboardSms m.id personId res.bookingId
```

Comment:
Can you rename it to `sendBookingOTPMessage`? What is dashboard specific in this?

Reply:
renamed